common/beerocks/bcl/include/beerocks/bcl/beerocks_message_structs.h

```diff
@@ -21,7 +21,7 @@ typedef struct {
     uint8_t src_bridge_mac[net::MAC_ADDR_LEN] = {};
     uint16_t length                           = 0;
     uint8_t swap_needed                       = 0;
-} sUdsHeader;
+} __attribute__((packed)) sUdsHeader;
```

Comment:
Note that this does not actually work (i.e. it doesn't necessarily get packed) because it non-POD because it uses the =0 initialisation. It happens to work correctly as long as the struct is not embedded in anything else (struct or array), but it's something to keep in mind...